partner-api.yaml

```diff
@@ -506,12 +513,91 @@ paths:
           description: Forbidden
         '404':
           description: Not Found
-
+  '/partner/{partnerId}/zugang':
+    parameters:
+      - schema:
+          type: string
+        name: partnerId
+        in: path
+        required: true
+        description: ID des Partners
+    get:
+      summary: Your GET endpoint
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ZugangPerson'
+        '403':
+          description: Forbidden - kein Zugriff
+        '404':
+          description: Not Found - Partner nicht gefunden
+      operationId: get-partner-partnerId-zugang
+      description: Liefert die Zugangsdaten eines Partners. Bei einer Person werden die Benutzer-Zugangsdaten und bei einer Organisation die Daten des Identity Provider geliefert.
+    post:
+      summary: Zugang anlegen
+      tags:
+        - Partner
+      operationId: post-partner-partnerId-zugang
+      responses:
+        '200':
+          description: OK - Zugang angelegt
+        '400':
+          description: Bad Request - Anfrage ungültig/konnte nicht verstanden werden
+        '406':
+          description: Not Acceptable
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                usernameNotAllowed:
+                  value:
+                    message: Username not allowed
+                    statusCode: 406
+                    statusMessage: Dieser Benutzername existiert bereits in Europace
+                    timestamp: '2021-01-20T10:03:29.02'
+                    traceId: XABC-2333...
+                alreadyExists:
+                  value:
+                    message: Already Exists
+                    statusCode: 406
+                    statusMessage: Es existiert bereits ein Zugang
+                    timestamp: '2021-01-20T10:03:29.02'
+                    traceId: XABC-2333...
+                wrongPartnerType:
+                  value:
+                    message: wrongPartnerType
+                    statusCode: 406
+                    statusMessage: Ein Zugang kann nur an einer Person nicht einer Organisation angelegt werden.
+                    timestamp: '2021-01-20T10:03:29.02'
+                    traceId: XABC-122...
+        '500':
+          description: Internal Server Error
```

Comment:
Nicht wirklich wichtig, aber 500er Fehler müssen wir nicht als Teil der SST dokumentieren. Diese sind ja bereits Teil des HTTP Protokolls. In der Service Definition sollten fachliche Fehler dokumentiert werden. Ich würde diesen Part löschen.